pkg/policies/policies.go

```diff
@@ -85,68 +97,17 @@ func (pv *PolicyVerifier) Verify(ctx context.Context) ([]*engine.PolicyViolation
 		pv.state.Attestation.Policies = append(pv.state.Attestation.Policies, &v12.Policy{
 			Name:       spec.Metadata.Name,
 			Attachment: policyAtt,
-			Body:       string(script.Source),
 			Violations: policyViolationsToAttestationViolations(res),
```

Comment:
we should probably also store when a policy was successfully ran not only when it's a violation, what do you think?

Reply:
In that case `Violations` will be empty, but the policy will still be there.